### DISCUSSION TITLE
Add the "save_begin" parameter to control the starting point (epoch or iteration) for saving checkpoints. 添加 save_begin 参数控制开始保存 checkpoint 的 epoch 数或者 iteration 数

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#472] [MMEngine]
# Add the "save_begin" parameter to control the starting point (epoch or iteration) for saving checkpoints. 
 添加 save_begin 参数控制开始保存 checkpoint 的 epoch 数或者 iteration 数 

Add a new parameter named "save_begin" for CheckpointHook to control the starting point (epoch or iteration) for saving checkpoints. 
 
 For more information and discussions about this feature, please refer to the following link: [[https://github.com/open-mmlab/mmengine/issues/1192](https://github.com/open-mmlab/mmengine/issues/1192)](https://github.com/open-mmlab/mmengine/issues/1192](https://github.com/open-mmlab/mmengine/issues/1192)) 

在CheckpointHook中添加一个名为save_begin的参数，以控制开始保存 checkpoint 的 epoch 数或者 iteraton 数。
 
 相关链接：[[https://github.com/open-mmlab/mmengine/issues/1192](https://github.com/open-mmlab/mmengine/issues/1192)](https://github.com/open-mmlab/mmengine/issues/1192](https://github.com/open-mmlab/mmengine/issues/1192)) 

## Task Information

**Repository**：MMEngine

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：KerwinKai

**Claim Time**：2023年07月24日

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1271

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by KerwinKai** [comment]
Hi, here is my pr:https://github.com/open-mmlab/mmengine/pull/1271
